sunpy/map/mapbase.py

```diff
@@ -10,19 +10,21 @@
 from base64 import b64encode
 from tempfile import NamedTemporaryFile
 from collections import namedtuple
+from packaging import version
 
 import matplotlib.pyplot as plt
 import numpy as np
 from matplotlib import cm
 from matplotlib.backend_bases import FigureCanvasBase
 from matplotlib.figure import Figure
 
+import astropy
 import astropy.units as u
 import astropy.wcs
 from astropy.coordinates import Latitude, Longitude, SkyCoord, UnitSphericalRepresentation
 from astropy.nddata import NDData
 from astropy.visualization import AsymmetricPercentileInterval, HistEqStretch, ImageNormalize
-from astropy.visualization.wcsaxes import WCSAxes
+from astropy.visualization.wcsaxes import WCSAxes, Quadrangle
```

Comment:
You need to protect such an import of `Quadrangle` with a `try..except`.

Reply:
Given my comment below (https://github.com/sunpy/sunpy/pull/4809#discussion_r551387434), I suggest removing all of your added global imports and just import locally within `draw_quadrangle()`